libs/components/tabs/src/lib/modules/tabs/tab-button.component.ts

```diff
@@ -127,7 +127,7 @@ export class SkyTabButtonComponent implements AfterViewInit, OnDestroy {
     this.#tabsetService.focusedTabBtnIndex
       .pipe(distinctUntilChanged(), takeUntil(this.#ngUnsubscribe))
       .subscribe((focusedIndex) => {
-        if (focusedIndex === this.tabIndex) {
+        if (focusedIndex.toLocaleString() === this.tabIndex?.toLocaleString()) {
```

Comment:
This change to add `toLocaleString` was needed because `focusedIndex` was a string; however, `this.tabIndex` is of type `number | string | undefined` and so we can't do just a straight comparison as the types don't match. The `toLocaleString` allows us to do that comparison.

Reply:
I'm not crazy about converting numbers to strings to compare them, especially when using `toLocaleString()` which introduces formatting (decimal separators, thousands separators, etc). What about this? This should preserve the previous behavior as long as `focusedIndex` is never `undefined`:

```
if (this.tabIndex !== undefined && focusedIndex === this.tabIndex) {
```